hack/aks/Makefile

```
@@ -95,6 +95,24 @@ byocni-up: swift-byocni-up ## Alias to swift-byocni-up
 cilium-up: swift-cilium-up ## Alias to swift-cilium-up
 up: swift-up ## Alias to swift-up
 
+
+nodesubnet-cilium-up: rg-up overlay-net-up ## Brings up an Overlay BYO CNI cluster without kube-proxy
```

```suggestion
nodesubnet-byocni-nokubeproxy-up: rg-up overlay-net-up ## Brings up an NodeSubnet BYO CNI cluster without kube-proxy
```

We are not bringing up a managed cilium cluster, but a byocni. Also, I don't believe this is using any overlay componets..?